svir/dialogs/viewer_dock.py

```diff
@@ -471,9 +475,13 @@ def set_output_type_and_its_gui(self, new_output_type):
     def extract_npz(
             self, session, hostname, calc_id, output_type, params=None):
         url = '%s/v1/calc/%s/extract/%s' % (hostname, calc_id, output_type)
-        resp_content = session.get(url, params=params).content
-        if not resp_content:
-            raise ValueError('GET %s returned an empty content!' % url)
+        resp = session.get(url, params=params)
+        if not resp.ok:
+            msg = "Unable to extract %s with parameters %s: %s" % (
+                output_type, params, resp.reason)
+            log_msg(msg, level='C', message_bar=self.iface.messageBar())
+            return
+        resp_content = resp.content
         return numpy.load(io.BytesIO(resp_content))
```

Comment:
You should a check for the case of missing content, otherwise an unclear error message like

```python
     File "/home/michele/oq-irmt-qgis/svir/dialogs/viewer_dock.py", line 475, in extract_npz
       return numpy.load(io.BytesIO(resp_content))
     File "/usr/lib/python2.7/dist-packages/numpy/lib/npyio.py", line 416, in load
       "Failed to interpret file %s as a pickle" % repr(file))
   Failed to interpret file <_io.BytesIO object at 0x7fc5d812b830> as a pickle
```

will appear